src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts

```diff
@@ -86,7 +86,7 @@ class spatialQueryMap extends viewModelBase {
             { color: spatialPolygonModel.colors[index % spatialPolygonModel.colors.length] }));
 
         const circleArray = this.circlesLayer().map((circle, index) => L.circle([circle.latitude, circle.longitude],
-            { color: spatialCircleModel.colors[index % spatialCircleModel.colors.length], fillOpacity: 0.2, radius: circle.radius }));
+            { color: spatialCircleModel.colors[index % spatialCircleModel.colors.length], fillOpacity: 0.4, radius: circle.radius }));
```

Comment:
imho - fillOpacity 0.4 is too much - reduce back to 0.2

Reply:
Reduced